### PR TITLE
feat:added login page to E-commerce

### PR DESCRIPTION
**Pull request by mariyasada**
1. added login page link to all files
2. added login page

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -45,10 +46,25 @@ a{
...
+    border: 4px solid var(--light-rose);
+    background: transparent;
 }
 .btn-wishlist
 {
     cursor: pointer;
 }
+.card-image
+{
+    object-fit: fill;
```
Comment (paarthbajaj) [inline main.css]: I will suggest using cover for the object-fit property, apply once, and see the changes yourself.

**Review by paarthbajaj — APPROVED** [review]

File: main.css (hunk 1 above)
Comment (mariyasada) [reply main.css]: okay

**Review by krishnakant01 — APPROVED** [review]

File: stylesheets/logIN/login.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    align-items: center;
+    justify-content: center;
+}
+.eye-icon
+{
+    position: absolute;
+    width:2rem;
+    height: 2rem;
+    border-radius: 0.2rem;
+    display: flex;
```
Comment (SaurabhChawda) [inline stylesheets/logIN/login.css:42]: For less than 1 rem you can use pixel

File: stylesheets/logIN/login.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    border: none;
+    background: transparent;
+    font-size: 2rem;
+}
+.input-textbox:focus{
+    border: 1px solid var(--red-color);
+}
+.link-btn.fluid-btn
+{
+   min-width: 31vw;
```
Comment (SaurabhChawda) [inline stylesheets/logIN/login.css:56]: Avoid using VW, use rem as much as possible.

**Review by SaurabhChawda — APPROVED** [review]

File: stylesheets/logIN/login.css (hunk 3 above)
Comment (mariyasada) [reply stylesheets/logIN/login.css:56]: okay

**Pull request merged**